### ISSUE TITLE
Lokole 0.6.11 & 0.6.12 released, but pip install not yet available [0.6.13, 0.6.14, 0.6.15, 0.6.16 & 0.8.0 have restored the pip install option. Need to finalize OPWEN variables' settings]

### ISSUE THREAD

**Issue by holta** [iiab/iiab#2767]
Thanks @nzola for this install attempt readout:

```
Running WifiSetup
Traceback
pi@raspberrypi:~ $ curl -fsO https://raw.githubusercontent.com/ascoderu/lokole/master/install.py && (most recent call last):
  File "install.py", line 933, in <module>
    cli()
  File "install.py", line 929, in cli
    main(parser.parse_args(), parser.error)
  File "install.py", line 797, in main
    wifi_setup()
  File "install.py", line 83, in _call_
    result = self._run()
  File "install.py", line 274, in _run
    self._configure_wifi()
  File "install.py", line 349, in _configure_wifi
    self.sh('systemctl start hostapd.service')
  File "install.py", line 202, in sh
    raise Exception(stderr)
Exception: Job for hostapd.service failed because the control process exited with error code.
See "systemctl status hostapd.service" and "journalctl -xe" for details.
pi@raspberrypi:~ $
```

@adamsclafani & @c-w, do you know if Lokole 0.6.11 might possibly be available at https://pypi.org/project/opwen-email-client/ for IIAB in coming days?

( Based on https://github.com/ascoderu/lokole/releases/tag/0.6.11 )

**Comment by jvonau** [comment]
When did Lokole gain the ability to control network services?

**Comment by jvonau** [comment]
> pi@raspberrypi:~ $ curl -fsO https://raw.githubusercontent.com/ascoderu/lokole/master/install.py &&

Is this installation routine being tested outside of IIAB?
https://github.com/ascoderu/lokole/blob/master/install.py#L261-L265
Those 2 services would be installed before the role is reached in IIAB. Think this testing is missing some prerequisites before starting the install of Lokole.

**Comment by nzola** [comment]
@jvonau Yes Jerry this is being tested outside of IIAB.

**Comment by nzola** [comment]
@holta Please add Shaun Bathgate (me@shaunb.ca) in this conversation. He is the one who is volunteering on PR merge. @sbathgate

**Comment by nzola** [comment]
LOL! I cannot win. Shaun was one of the Canadian's Navy..He knows the in
and out .LOL

**Comment by holta** [comment]
Huge Thanks to @sbathgate and @c-w for the server-side work on [Lokole 0.6.12](https://github.com/ascoderu/lokole/releases/tag/0.6.12) released overnight!

1) Will PR https://github.com/ascoderu/lokole/pull/512 (Implement Terraform) also be necessary?

2) What's your general recommendation on the client side ?
(e.g. https://pypi.org/search/?q=opwen_email_client does not show a release since July 2020's 0.6.10)

_Thank You to Everyone (including @adamsclafani) for your very hard work here &mdash; as everyone's now trying hard to bring this Lokole + IIAB integration to completion!_

Refs:

- ascoderu/lokole#454 "Enable email sync worker on IIAB"
- PR #2773 "Testing latest lokole - use lokole_commit for install"

**Comment by holta** [comment]
Lokole 0.6.13 (2 days ago) and 0.6.14 (yesterday) were both released with the pip install option restored!  As can be seen here: https://pypi.org/project/opwen-email-client/

- Should we close this ticket now?
- What's the best place to open a new ticket to track general progress here?

_Thanks @jvonau and **everybody** for the heroic + ongoing efforts!_

PS @deldesir in Haiti would like to assist in testing as this begin to converge for actual field/community use.

Ref: https://github.com/ascoderu/lokole/issues/546

**Comment by jvonau** [comment]
I'd file any new issues directly upstream. Keep this one open for what the OPWEN variables should be set to for IIAB purposes from #2773 and dumping apache #2507 that @deldesir might be interested in.

**Comment by jvonau** [comment]
Rename the title of this issue to reflect tracking environment variables used by lokole as I'm prohibited... smh..

**Comment by nzola** [comment]
I think naming or renaming is Adam H. job?

**Comment by holta** [comment]
See PR #2789 "OPWEN_SETTINGS" where progress is happening quickly.

**Comment by holta** [comment]
See PR #2790 "Lok services" where progress is continuing thanks to @jvonau & @adamsclafani.

**Comment by holta** [comment]
Please see PR #2800 "Lok super" which evolved from PR #2792 "Lok user"

Progress is continuing thanks to @jvonau & @adamsclafani.

A call will be held at 12 Noon PDT Tuesday (tomorrow) to review what is changing, including the upcoming [Lokole](https://github.com/ascoderu/lokole) 0.6.16 expected soon.

**Comment by nzola** [comment]
Sure! Looking forward to...

**Comment by holta** [comment]
Please see PR #2802 "Lok merge" which is a trimmed down version of PR #2800 "Lok super"

**Comment by nzola** [comment]
Shall I do ./runrole --reinstall lokole

**Comment by holta** [comment]
> Shall I do ./runrole --reinstall lokole

A more complete test would be a clean/fresh OS install including PR #2802:

```
curl d.iiab.io/install.txt | sudo bash -s 2802
```

But consult with @jvonau first if possible.

**Comment by nzola** [comment]
ok

**Comment by jvonau** [comment]
@nzola use `curl d.iiab.io/install.txt | sudo bash -s 2800` on your new installs going forward, remember to create local_vars.yml set with
```
lokole_sim_type: 
lokole_client_id:
```

**Comment by nzola** [comment]
Thanks Adam!

**Comment by holta** [comment]
> Thanks Adam!

FYI that was Jerry not me (:

**Comment by nzola** [comment]
Thanks much Jerry!

**Comment by jvonau** [comment]
@nzola Once https://github.com/ascoderu/lokole/pull/548 is pushed, the install instructions need a little altering:
Use just `curl d.iiab.io/install.txt | sudo bash`
Only use `lokole_sim_type: ` DO NOT USE `lokole_client_id:`

**Comment by nzola** [comment]
@Jerry Vonau ***@***.***> Ok got it. Is it pushed yet? Thanks!

**Comment by jvonau** [comment]
@nzola `sudo /library/lokole/venv/bin/manage.py createadmin --name=admin --password=changeme`

**Comment by jvonau** [comment]
close
